apps/frontend/src/shared/i18n/locales/he/dialogs.json

```
@@ -0,0 +1,223 @@
+    "deleteTitle": "למחוק עץ עבודה?",
+    "deleteDescription": "פעולה זו תמחק לצמיתות את עץ העבודה ואת כל השינויים שלא הועברו ל-commit.",
+    "cannotUndo": "לא ניתן לבטל פעולה זו.",
+    "deleting": "מוחק..."
+  },
+  "update": {
+    "title": "Auto Claude",
+    "projectInitialized": "הפרויקט אותחל."
+  },
+  "appUpdate": {
+    "title": "עדכון אפליקציה זמין",
+    "description": "גרסה חדשה של Auto Claude מוכנה להורדה",
```

Biome (2.1.2)

[error] The key worktrees was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)